src/main/java/htsjdk/samtools/liftover/LiftOver.java

```diff
@@ -54,6 +54,33 @@
     private final OverlapDetector<Chain> chains;
     private final Map<String, Set<String>> contigMap = new HashMap<>();
 
+    private boolean logFailedIntervals = true;
+    private long totalFailedIntervalsBelowThreshold = 0L;
+
+    /**
+     * By default any lifted interval that falls below liftOverMinMatch
+     * will be logged.  Set this to false to prevent logging.
+     * @param logFailedIntervals
+     */
+    public void setLogFailedIntervals(boolean logFailedIntervals) {
```

Comment:
what is this method for? 

could you replace it with "increment..." and use that function instead of ++ below?

Reply:
I'm not sure I follow this.  The intent of this PR is to let callers of LiftOver adjust the logging behavior.  This isnt a setter for the counter itself (though I can see the confusing name choice - more of setDoLogFailedIntervals().  A different method name would be fine).  This method allows callers to turn off logging every single time an interval is rejected.  

It is not intended to be some kind of method for addFailedInterval(), which I think your second comment might be aimed toward?

Reply:
yes. sorry, I got confused.

Reply:
perhaps setShouldLogFailedIntervals ?